p/cmta/example.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const createClient = require('../..')
+const cmtaProfile = require('.')
+
+const client = createClient(cmtaProfile)
+
+// Broadie Oaks to Domain
+client.journeys('000002370', '000005919', {results: 1, polylines: true})
+// client.departures('000002370', {duration: 1})
+// client.locations('Westgate', {results: 2})
+// client.location('000005534') // Downtown light rail station
+// client.nearby(30.266222, -97.746058, {distance: 60})
```

Comment:
This should be an object. Check [the docs for `nearby()`](https://github.com/public-transport/hafas-client/blob/f3d8304db879ce54fef0e0a8daa1ef0a5e292c0b/docs/nearby.md).